### ISSUE TITLE
DICOM to BIDS conversion

### ISSUE THREAD

**Issue by mih** [datalad-handbook/book#135]
This is likely to most common use case in neuroimaging. So it would be highly desirable to have this covered. @bpoldrack 's docs for `datalad-hirni` could serve as a starting point.

In terms of connecting this with the primary content of the book, we would need to introduce the concept of a datalad *extension package*, because hirni uses quite a few of them.

**Comment by bpoldrack** [comment]
I'm a bit skeptical whether this makes sense to cover as a use-case, since there's a lot to customize in `datalad-hirni` and it's quite unlikely that it works in the simplest form out-of-the-box for some user data. (at least currently - there's things to be done that could possibly provide way better defaults) So, the message likely would be like: 

"Look it could be this simple if only you knew how it works or have someone else who does and provides some institutional setup/configuration. Here you have an example dataset to reproduce the simplicity but chances are you will fail to get that from your own data.
And now, please read the documentation of hirni for all the details."

I might be to pessimistic about that, but this is what I'd expect to be the result of just showing the conversion demo. And covering hirni in it's entirety is really a chapter not a use-case description it seems to me. May be there's a good way to cover it, but that's not easy to come up with. The conversion itself is a single call that wouldn't explain anything. To get a better grasp one needs the notion of a study dataset in addition and an idea of what a specification looks like, etc.

ATM I think it might be better to approach it the other way around: Have an introduction to `datalad-hirni` in the "style" of this handbook in hirni's docs and link it here as an example of an extension.

**Comment by mih** [comment]
I agree that the package itself needs better docs, and other improvements. However, in the scope of the book we don't want documentation on hirni (and surely not on everything it could do). Instead, we would focus on the principle idea: Have a non-normalized data structure that uses metadata to fill all gaps in order to enable automated processing/normalization (later on). The use cases part is meant to give people insight on how the existing tooling can be used to solve particular problems. Sometimes that won't be possible with a three-line script, but that doesn't make it useless IMHO.

**Comment by bpoldrack** [comment]
FTR: Then https://datalad-hirni.readthedocs.io/en/latest/demos.html is the place to look at for getting started with such a use-case description.

**Comment by mih** [comment]
Perfect, thx!

**Comment by adswa** [comment]
Thanks a lot, @bpoldrack!

**Comment by PeerHerholz** [comment]
The [heudiconv tutorials](https://heudiconv.readthedocs.io/en/latest/tutorials.html) might also be a good place to have a look. However, it's of course not focused on `DataLad` but integrates it, as well as utilizes it for data discovery and download. Additionally, the same problems outlined by @bpoldrack will arise in that folks will have difficulties adapting for their own data.
